packages/frontend-2/components/global/icon/Connectors.vue

```diff
@@ -0,0 +1,16 @@
+<template>
```

Comment:
Should we update this in the user nav as well so they are the same icon?

Reply:
The icons were dropped from the user nav a while ago:
<img width="270" alt="image" src="https://github.com/user-attachments/assets/676caa6a-0dc5-4e15-bba5-f3e8c193abfe">

Unless you meant a different menu?

Reply:
Sorry you are right, in my mind it still had that cloud cloud icon haha